htdocs/lang/de/ocstyle/main.tpl.php

```diff
@@ -344,5 +344,41 @@ function flashbutton(bname)
             </div> <!-- page-container-1 -->
 
         </div> <!-- overall -->
+        {literal}
+        <script type="text/javascript">
+            // Set to the same value as the web property used on the site
+            var gaProperty = 'UA-59334952-4';
```

Comment:
Die Property ist eh "öffentlich" einsehbar. Wollte das in der neuen Theme-Struktur anpassen :)